.github/workflows/c1-v2.yml

```
@@ -0,0 +1,190 @@
+          cp -r deploy/nginx deploy-package/
+          cp -r deploy/server deploy-package/
+          
+          # compose가 load 할 .env package
+          echo "IMAGE_TAG=${{ github.sha }}" > deploy-package/.env
+
+      - name: Upload to S3 and deploy
+        run: |
+          cd deploy-package
+          zip -r ../deploy.zip .
+          cd ..
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)